include/api/CDataFrameOutliersRunner.h

```diff
@@ -32,6 +32,11 @@ class API_EXPORT CDataFrameOutliersRunner final : public CDataFrameAnalysisRunne
     CDataFrameOutliersRunner(const CDataFrameAnalysisSpecification& spec,
                              const CDataFrameAnalysisParameters& parameters);
 
+    //! \return Reference to the analysis state.
+    const CDataFrameAnalysisState& state() const override;
```

Comment:
As above, I'd consider some different naming for these things, i.e. maybe `status`.